src/test/java/com/epam/tc/hw3/bonus/fluent/tests/FluentTestElementsPageAbstract.java

```diff
@@ -0,0 +1,51 @@
+package com.epam.tc.hw3.bonus.fluent.tests;
+
+import com.epam.tc.hw3.bonus.fluent.pages.FluentDifferentElementsPageObject;
+import com.epam.tc.hw3.bonus.fluent.pages.FluentHomePageObject;
+import com.epam.tc.hw3.tests.AbstractPagesUITest;
+import com.epam.tc.hw3.utils.PagesDataProviders;
+import java.util.Map;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+public class FluentTestElementsPageAbstract extends AbstractPagesUITest {
+
+    private FluentHomePageObject homePage;
+    private FluentDifferentElementsPageObject elementsPage;
+
+    @BeforeClass
+    void navigateToHomePage() {
+        homePage = new FluentHomePageObject(webDriver);
+    }
+
+
+    @Test(dataProvider = "loginData", dataProviderClass = PagesDataProviders.class)
+    void login(String login, String password, String expectedUserName) {
+        homePage.signIn(login, password).checkSignIn(expectedUserName);
+    }
+
+
+    @Test(dependsOnMethods = {
```

Comment:
Try to avoid dependencies on other tests tests

Reply:
Done